katello/katello-backup

```diff
@@ -1,94 +1,113 @@
 #!/bin/bash
 
 usage(){
-  echo "Usage: katello-backup <backup directory>"
-  echo " eg: $ katello-backup /tmp/katello-backup/"
+  echo "Usage: katello-backup --directory <backup directory>"
+  echo " eg: $ katello-backup --directory /tmp/katello-backup/"
```

Comment:
They would follow after the directory argument. It's doable, this is easier for me and opens it up for user, but I can change it do katello-backup <directory> --option1 --option2 if you feel that's nicer.